mllib/src/main/scala/org/apache/spark/mllib/linalg/Matrices.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.mllib.linalg
 
-import java.util.{Arrays, Random}
+import java.util.{Arrays => jArrays, Random}
```

Comment:
Any reason to change this? Scala doesn't have `Arrays`. If we want to emphasize this is from Java, we can do `import java.{util => ju}` and use `ju.Arrays` in the code. This is recommended by @srowen

Anyway, it is strange to have a class name alias start with a lowercase letter.